locksmith/examples/websub_discord/src/server.ts

```diff
@@ -0,0 +1,63 @@
+import { WebhookClient, MessageEmbed } from 'discord.js'
+import express from 'express'
+import { config } from './config'
+import { chunk } from './util'
+import { createWebsubMiddleware } from './middleware'
+
+const port = process.env.PORT || 4000
+
+const webhookClient = new WebhookClient(config)
+
+const websubMiddleware = createWebsubMiddleware({
+  secret: config.websubSecret,
+})
+
+const app = express()
+app.use(express.json())
+
+app.post('/callback/locks', websubMiddleware, async (req) => {
+  const embeds: MessageEmbed[] = []
+  const locks: any[] = req.body?.data
+  if (!locks.length) {
+    return
+  }
+  for (const lock of locks) {
+    const embed = new MessageEmbed()
+    embed.setTitle('New Lock')
```

Comment:
I think it would be cool to add the network as well? So we know on what network this new lock is? And ideally a link to the block explorer for the network?

Reply:
I'm going to change this part later. It's just to see if things work correctly for testing purposes right now.

My plan for this is to eventually become something that people can easily self host or we can host it and have others provide us with their discord webhook creditionals to push the updates. 

This PR is already getting big as it is.